src/doc_builder/commands/preview.py

```diff
@@ -175,9 +175,9 @@ def preview_command(args):
             kit_routes_folder = tmp_dir / "kit" / "src" / "routes"
             # files/folders cannot have a name that starts with `__` since it is a reserved Sveltekit keyword
             for dir in output_path.glob("**/__*/*"):
-                dir.rmdir()
+                subprocess.run(["rm", "-rf", str(dir)])
```

Comment:
```suggestion
                shutil.rmtree(dir)
```
This is cleaner.

Reply:
It only works for dir, so you should make a test. It's still better than using `subprocess` for this.

Reply:
handled in https://github.com/huggingface/doc-builder/pull/139/commits/635938b0ec5030013266dc2d8299d4743056da52